### PR TITLE
[General] Improvements (bug fixes, tests)

### PR DESCRIPTION
**Pull request by gpotter2**
@ashiina This PR:
- fixes https://github.com/ashiina/lambda-local/issues/84
- fixes https://github.com/ashiina/lambda-local/issues/82
- adds tests for timeout
- add tests for the bin/ directly ==> avoid more critical bugs :)

### CODE REVIEW

File: test/other/debug.aws [hunk 1]
```diff
@@ -1,7 +1,8 @@
 [default]
 something_other=sdfsdf
-aws_access_key_id=AKIAIOSFODNN7EXAMPLE
+aws_access_key_id =      AKIAIOSFODNN7EXAMPLE
```
Comment (pselle) [inline test/other/debug.aws]: This looks like it could be unintented

File: test/test.js [hunk 2]
```diff
@@ -170,3 +206,28 @@ describe("- Testing lambdalocal.js", function () {
...
+        });
+    });
+    describe("* Failing Run", function () {
+        it("should fail", function () {
+            var command = get_shell("node ../bin/lambda-local -l ./functs/test-func-fail.js -e ./events/test-event.js");
+            var r = spawnSync(command[0], command[1]);
+            assert.equal(r.status, 1);
+        });
+    });
+    describe("* Timeout Run", function () {
```
Comment (pselle) [inline test/test.js]: Yay timeout test!

File: test/other/debug.aws (hunk 1 above)
Comment (gpotter2) [inline test/other/debug.aws]: No that's on purpose: the three parameters are using very different spaces in order to test that the detection works properly :)

**Comment by AWare** [comment]
I'm also experiencing #82 and this PR resolves it for me.
(Thanks @gpotter2 )

**Comment by edthorne** [comment]
@ashiina will this be released soon?

**Comment by gpotter2** [comment]
Hello, as this is getting ridiculous, i would recommand you to use https://github.com/gpotter2/lambda-local instead... I'll be updating the project with the different PRs on this repo.

**Pull request closed without merging**